enclave/src/test_polkadex_gateway.rs

```diff
@@ -297,7 +297,7 @@
 // //         user_uid: buy_order_user.clone(),
 // //         market_id: MarketId {
 // //             base: AssetId::POLKADEX,
-// //             quote: AssetId::DOT,
+// //             quote: AssetId::Asset(0),
```

Comment:
Why this large chunk of double commanded code?

Should be removed or made to work.

Reply:
I don't think I should make this PR any bigger with changes such as this. This should be done in a separate PR, should it not?